src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -118,8 +125,16 @@
   (when-not (string? (:path (:match-request rule)))
     (throw (IllegalArgumentException.
             (str "The path set in the authorization rule specified as "
-                 (pprint-rule (:path (:match-request rule))) " is invalid. "
+                 (pprint-rule rule) " is invalid. "
                  "It should be a string."))))
+  (when (and (not (nil? (:method rule)))
```

Comment:
(facepalm) you're right, it won't work for a vector. And I thought it was above the match-request because I was look at the config example in the "Trapperkeeper Authorization Design Doc." Which was probably not a good idea.